net-mgmt/pfSense-pkg-Telegraf/files/usr/local/pkg/telegraf.inc

```diff
@@ -51,12 +51,31 @@ function telegraf_resync_config() {
 		$telegraf_conf['interval'] = 10;
 	}
 
+	$shortname = '';
+	if ($telegraf_conf['shortname'] == 'on') {
+		$shortname = "\thostname = \"" . $config['system']['hostname'] . "\"";
+	}
+
+	$ca_pem_file = "/usr/local/etc/telegraf.ca";
+	$a_ca = $config['ca'];
```

Comment:
There is no need to make a copy of `$config['ca']` just to use it on below foreach(). It's better to just use it instead

Reply:
I adapted the code style from official pfsense source code like you see in the following two references:

https://github.com/pfsense/pfsense/blob/7c41a378ee282fb43b285b66283160686e7cb5ca/src/usr/local/www/vpn_openvpn_client.php#L51

https://github.com/pfsense/pfsense/blob/7c41a378ee282fb43b285b66283160686e7cb5ca/src/usr/local/www/vpn_openvpn_client.php#L685